### PR TITLE
Use pnpm instead of yarn

### PR DESCRIPTION
**Pull request by philipsens**
Hello all,

I want to suggest to start using pnpm. This has many advantages. Like being able to share the node_modus folder across all projects, significantly shrinking disk space. It also has a more readable lock file, which should have fewer problems with conflicts.

The (Jenkins & GitHub) CI has already adapted to make use of pnpm. Being able to cache the global node_modules will improve build times on Jenkins as well.

Pnpm is packages with corepack (like yarn) and can be enabled by typing:
```shell
corepack enable pnpm
```

I hope you all understand my incentives.

### CODE REVIEW

File: .github/workflows/testing.js.yml [hunk 1]
```diff
@@ -43,49 +43,51 @@ jobs:
...
+        with:
+          path: ${{ env.STORE_PATH }}
+          key: ${{ runner.os }}-pnpm-store-${{ hashFiles('**/pnpm-lock.yaml') }}
+          restore-keys: |
+            ${{ runner.os }}-pnpm-store-
 
-      - name: Install packages
-        run: yarn install --immutable
+      - name: Install dependencies
+        run: pnpm install
```
Comment (MatthijsSmets) [inline .github/workflows/testing.js.yml:74]:
```suggestion
        run: pnpm install --frozen-lockfile
```

Comment (philipsens) [reply .github/workflows/testing.js.yml:74]: couldn't find the file

Comment (philipsens) [reply .github/workflows/testing.js.yml]: lets try again

Comment (philipsens) [reply .github/workflows/testing.js.yml]: works now

File: .github/workflows/testing.js.yml [hunk 2]
```diff
@@ -43,49 +43,51 @@ jobs:
...
-        working-directory: "ladybug-frontend"
-      - name: Show npm version
-        run: npm -v
-        working-directory: "ladybug-frontend"
-      - name: Show node version
-        run: node -v
-        working-directory: "ladybug-frontend"
-      - name: Show yarn version
-        run: yarn -v
-        working-directory: "ladybug-frontend"
```
Comment (mhdirkse) [inline .github/workflows/testing.js.yml:69]: Hier worden actions weggehaald die versienummers van node/npm/yarn ed tonen. Kan je actions behouden om van al dit soort tools het versienummer te tonen?

**Review by mhdirkse — DISMISSED** [review]
In principe goed. Eén klein puntje.

File: .github/workflows/testing.js.yml (hunk 2 above)
Comment (philipsens) [reply .github/workflows/testing.js.yml:69]: Het wordt al gelogd
image

**Review by MatthijsSmets — APPROVED** [review]

**Pull request merged**